README.md

```diff
@@ -2,7 +2,7 @@
 
 This example app serves as an example of how one can easily instrument HTTP handlers with [Prometheus][prometheus] metrics. It uses the Prometheus [go client][client-golang] to create a new Prometheus registry.
 
-Usage is simple, on any request to `/` the request will result in a `200` response code. This increments the counter for this response code. Similarly the `/err` endpoint will result in a `404` response code, therefore increments that respective counter.
+Usage is simple, on any request to `/` the request will result in a `200` response code. This increments the counter for this response code. Similarly the `/err` endpoint will result in a `404` response code, therefore increments that respective counter. Duration metrics are also exposed for any request to `/`.
 
 A Docker image is available at: `quay.io/brancz/prometheus-example-app:v0.1.0`
```

Comment:
Hmm we should probably build the image at a later point when we implement all the metrics we want.

But this should be quay.io/brancz/prometheus-example-app:v0.2.0, but lets fix that in a follow up PR as its not related to this.

Reply:
Yes I noticed the version issue, was fixing it in another branch